apps/meeting/package.json

```diff
@@ -15,11 +15,13 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-router-dom": "^5.1.2",
+    "rimraf": "^3.0.2",
     "styled-components": "^5.0.0",
     "styled-system": "^5.1.5",
     "throttle-debounce": "^2.3.0"
   },
   "scripts": {
+    "postinstall": "node scripts/update-react-library.js",
```

Comment:
Is there a reason to use postinstall vs preinstall?

Reply:
Any works, just wanted to do it once the rest are installed. Do not think it really matters here, should I use pre instead?

Reply:
If I ever want to use the npm version, how would I do that?

If I do npm install amazon-chime-sdk-component-library-react@2.9.0

wouldnt the postinstall script run and just overwrite the amazon-chime-sdk-component-library-react dependency to point to github master?

Reply:
Yeah good question and I have tested it already. So, the `post` `pre` install scripts will only run on the `npm install` script and not `npm install <package>`.